draft-ietf-quic-multipath.md

```diff
@@ -702,14 +702,45 @@ maintaining an order list of packets sent on each path. That ordered
 list can then be used to compute acknowledgement gaps per path in
 Packet Threshold tests.
 
-### ACK Delay and Zero-Length CID Considerations
-
-The ACK Delay field of the ACK frame is relative to the largest
-acknowledged packet number (see {{Section 13.2.5 of QUIC-TRANSPORT}}).
-When using paths with different transmission delays, the reported host
-delay will most of the time relate to the path with the shortest latency.
-To collect ACK delays on all the paths, hosts should rely on time stamps
-as described in {{QUIC-Timestamp}}.
+### RTT Estimation Considerations when SPNS is Used {#ack-delay-and-zero-length-cid-considerations}
+
+When SPNS is in use, accurate RTT estimation requires more careful
+considerations. According to {{QUIC-RECOVERY}}, an endpoint generates an RTT
+sample on receiving an ACK frame that meets the following two conditions: (1)
+the largest acknowledged packet number is newly acknowledged, and (2) at least
+one of the newly acknowledged packets was ack-eliciting. The RTT sample,
+latest_rtt is calculated as the time elapsed since the largest acknowledged
+packet was sent. However, when applying the above algorithm with SPNS, one may
+encounter the following issues: (1) RTT of some paths are not updated timely if
+ACKs are mostly returned from other paths, and (2) ACK frames depend on the
+largest received packet of the connection, not the path, and the resulted RTT
+sample may be the sum of the uplink delay of one path and the downlink delay of
```

Comment:
... and the resulted RTT sample may be the sum of _the one way delay of two different paths._